draft-ldbc-cats-framework.md

```diff
@@ -315,14 +345,20 @@ Note that, depending on the design considerations and service requirements, per-
+The "underlay infrastructure" in {{fig-cats-components}} indicates an IP/MPLS network that is not necessarily CATS-aware. The CATS paths that are computed by a P-CS will be distributed among the CATS-Forwarders ({{sec-ocr}}), and will not affect the underlay nodes. Underlay nodes are typically P routers ({{Section 5.3.1 of ?RFC4026}}).
 
 ## Deployment Considerations
 
 This document does not make any assumption about how the various CATS functional elements are implemented and deployed. Concretely, whether a CATS deployment follows a fully distributed design or relies upon a mix of centralized (e.g., a C-PS) and distributed CATS functions (e.g., CATS traffic classifiers) is deployment-specific and may reflect the savoir-faire of the (CATS) service provider.
 
 Centralized designs where the computing related metrics from the C-SMAs are collected by a (logically) centralized path computation logic (e.g., a Path Computation Element (PCE) {{?RFC4655}}) that also collects network metrics may be adopted. In the latter case, the CATS computation logic may process incoming service requests to compute and select paths and, therefore, service contact instances. The outcomes of such a computation process may then be communicated to CATS traffic classifiers (C-TCs).
 
+In conclusion, considering implementation and deployment of CATS framework, e.g., distribution of service metrics and implementation of C-PS, three possible deployment models are listed below:
+* Distributed model: Computing metrics are distributed among network devices directly using distributed protocols without interactions with a centralized control plane. Service scheduling function is performed by the CATS routers in the distribution model, Therefore, the C-PS is integrated into the ingress CATS-Forwarder.
+* Centralized model: Computing metrics are collected by a centralized control plane, and then the centralized control plane performs service scheduling function, and computes the forwarding path for service requests and syncs up with the Ingress CATS-Forwarder. In this model, C-PS is implemented in the centralized control plane.
+* Hybrid model: A combination of Distribution and Centralized models. A part of computing metrics are distributed among the network devices, and others may be collected by a centralized control plane. For example, some static information such as capabilities information can be distributed among network devices since they are quite stable. For the frequent changing information such as utilization of resources, can be collected by a centralized control plane to avoid frequent flooding in distributed control plane. Service scheduling function can be performed by a centralized control plane and/or the CATS router.The entire or partial C-PS function may be implemented in the centralized control plane, depending on the specific implementation and deployment.
```

Comment:
```suggestion
* Hybrid model: is a combination of distribution and centralized models.
: A part of computing metrics are distributed among the network devices, and others may be collected by a centralized control plane. For example, some static information (e.g., capabilities information) can be distributed among network devices since they are quite stable. Frequent changing information (e.g., resource utilization) can be collected by a centralized control plane to avoid frequent flooding in the distributed control plane. Service scheduling function can be performed by a centralized control plane and/or the CATS forwarder. The entire or partial C-PS function may be implemented in the centralized control plane, depending on the specific implementation and deployment.
```